generate_test.R

```diff
@@ -0,0 +1,14 @@
+
+# generate test_result and compare to analytical results
+# all temp increases are set to zero
+# temp pulses are set to zero except for Angola which gets a temperature pulse of 1 degree in 2021
+
+# default option is RCP 4.5, SSP1, eta of 1 and bhm damage function
+# change to option to choose different values? But how to do so within a source function
+source(generate_cscc_test.R)
```

Comment:
Please don't duplicate the whole code for the sake of one line! At the moment you can just leave the default at test=true. Really we should rewrite this as a function which takes several arguments and run it with a separate file.

Reply:
Ah okay, and would that function just set those options?

Reply:
the function takes the arguments and returns the values you are interested in.